packages/app/src/server/service/page.ts

```diff
@@ -561,12 +561,6 @@ class PageService {
     const nToIncrease = (renamedPage.isEmpty ? 0 : 1) + page.descendantCount;
     await this.updateDescendantCountOfAncestors(renamedPage._id, nToIncrease, false);
 
-    // Remove leaf empty pages if not moving to under the ex-target position
-    if (!this.isRenamingToUnderTarget(page.path, newPagePath)) {
-    // remove empty pages at leaf position
-      await Page.removeLeafEmptyPagesRecursively(page.parent);
-    }
```

Comment:
We would like to keep this code.
Is there any solution other than removing this code?

Reply:
The method of `removeLeafEmptyPagesRecursively` has been returned